### PR TITLE
Add examples at overrides.md, customize resources and add variables from secretRef

### PR DESCRIPTION
**Pull request by DodgeCamaro**

### CODE REVIEW

File: docs/features/overrides.md [hunk 1]
```diff
@@ -208,4 +209,53 @@ spec:
...
+            template:
+              spec:
+                containers:
+                  - name: service
+                    envFrom:
+                      - secretRef:
+                          name: frontend
+```
+
+## Overrides container resources all services
```
Comment (alexandrevilain) [inline docs/features/overrides.md:237]: Could you please move this section to `Overrides for all services` and rename the title to something like `Example: Override containers resources` ?

**Review by alexandrevilain — CHANGES_REQUESTED** [review]

**Comment by alexandrevilain** [comment]
Thanks for the doc improvement ! 💯 Just added a little comment.

File: docs/features/overrides.md (hunk 1 above)
Comment (DodgeCamaro) [reply docs/features/overrides.md:237]: sure

**Review by alexandrevilain — APPROVED** [review]
Awesome, thanks!!

**Pull request merged**